[BUG]

**Describe the bug**
A clear and concise description of what the bug is.
Trying to delete a collection from the manage content view results in a 500 error

**To Reproduce**
Steps to reproduce the behavior, including the results:
1. Go to "Manage Collection"
2. Click "Delete" on a collection 
3. 500 error for remove html template

**Expected behavior**
Collection is removed or confirmation page would display

**Screenshots/Screen Recording**
```
I, [2025-12-05T19:18:22.987270 #1]  INFO -- : [8adaaaa9-dac7-46ca-abff-94af527efaa0]   Rendered layout layouts/avalon.html.erb (Duration: 2039.4ms | GC: 116.6ms)
I, [2025-12-05T19:18:22.987564 #1]  INFO -- : [8adaaaa9-dac7-46ca-abff-94af527efaa0] Completed 500 Internal Server Error in 2335ms (ActiveRecord: 4.0ms (3 queries, 0 cached) | LDP: 3853.8ms | Solr: 323.7ms | GC: 143.1ms)
E, [2025-12-05T19:18:22.992226 #1] ERROR -- : [8adaaaa9-dac7-46ca-abff-94af527efaa0]
[8adaaaa9-dac7-46ca-abff-94af527efaa0] ActionView::Template::Error (missing required parameter params[:bucket])
[8adaaaa9-dac7-46ca-abff-94af527efaa0] Caused by: ArgumentError (missing required parameter params[:bucket])
[8adaaaa9-dac7-46ca-abff-94af527efaa0]
Information for: ActionView::Template::Error (missing required parameter params[:bucket]):
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     28:     content_pronoun = @objects.count > 1 ? 'them' : 'it'
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     29:     %>
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     30:
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     31:     <% if @collection.dropbox_object_count > 0 %>
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     32:       <div class="alert alert-warning">
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     33:         <span class="fa fa-warning" aria-hidden="true"></span>
[8adaaaa9-dac7-46ca-abff-94af527efaa0]     34:         <span>There are saved files in this collection's <b>Dropbox</b> that will be deleted when this collection is deleted.</span>
[8adaaaa9-dac7-46ca-abff-94af527efaa0]
[8adaaaa9-dac7-46ca-abff-94af527efaa0] app/models/admin/collection.rb:222:in 'Admin::Collection#dropbox_object_count'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] app/views/admin/collections/remove.html.erb:31
[8adaaaa9-dac7-46ca-abff-94af527efaa0] config/initializers/ldp_solr_duration_logger.rb:35:in 'AvalonInstrumentation#cleanup_view_runtime'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] config/initializers/ldp_solr_duration_logger.rb:25:in 'AvalonInstrumentation#process_action'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] lib/tempfile_factory.rb:36:in 'TempfileFactory#call'
[8adaaaa9-dac7-46ca-abff-94af527efaa0]
Information for cause: ArgumentError (missing required parameter params[:bucket]):
[8adaaaa9-dac7-46ca-abff-94af527efaa0]
[8adaaaa9-dac7-46ca-abff-94af527efaa0] app/models/admin/collection.rb:222:in 'Admin::Collection#dropbox_object_count'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] app/views/admin/collections/remove.html.erb:31
[8adaaaa9-dac7-46ca-abff-94af527efaa0] config/initializers/ldp_solr_duration_logger.rb:35:in 'AvalonInstrumentation#cleanup_view_runtime'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] config/initializers/ldp_solr_duration_logger.rb:25:in 'AvalonInstrumentation#process_action'
[8adaaaa9-dac7-46ca-abff-94af527efaa0] lib/tempfile_factory.rb:36:in 'TempfileFactory#call'
```

**Environment (please complete the following information):**
 - Device / platform macbook pro
 - OS: osx / Terraform AWS Deployment
 - Browser Brave
 - MSU Terraform Deployment
 - Avalon Version 8.1.1

**Release:**
This is only an issue we've encountered since upgrading to 8.1.1

**Additional notes**

--- a/scripts/compose-init.sh
+++ b/scripts/compose-init.sh
@@ -122,6 +122,7 @@ SETTINGS__ENCODING__ENGINE_ADAPTER=${encoding_engine_adapter}
 SETTINGS__ENCODING__PIPELINE=${elastictranscoder_pipeline_id}
 SETTINGS__ENCODING__MEDIA_CONVERT_ROLE=${media_convert_role}
 SETTINGS__ENCODING__DERIVATIVE_BUCKET=${derivatives_bucket_id}
+SETTINGS__ENCODING__MASTERFILE_BUCKET=${masterfiles_bucket_id}
 SETTINGS__ENCODING__PRESETS_PATH=config/media_convert_presets/
 SETTINGS__EMAIL__COMMENTS=${email_comments}
 SETTINGS__EMAIL__NOTIFICATION=${email_notification}

